src/ansys/fluent/core/meshing/workflow.py

```diff
@@ -50,6 +41,11 @@ def _command(self):
         def __getattr__(self, attr):
             return getattr(self._task, attr)
 
+        def __dir__(self):
+            return sorted(
+                set(list(self.__dict__.keys()) + dir(type(self)) + dir(self._task))
```

Comment:
add this object's attributes, the class attributes (which brings in methods), and attributes inherited from the task